src/ui/drawing/ISurface.hh

```diff
@@ -23,12 +23,12 @@ public:
     /// <summary>
     /// Gets the width of the surface.
     /// </summary>
-    virtual size_t GetWidth() const = 0;
+    virtual std::ptrdiff_t GetWidth() const = 0;
```

Comment:
why not `size_t`?

Reply:
Because  size_t is not an appropriate type for dimensions or coordinates. Plus in all derived classes, the dimensions are signed.